lib/logstash/inputs/elasticsearch.rb

```diff
@@ -640,6 +678,29 @@ def setup_query_executor
                       end
   end
 
+  def setup_cursor_tracker
+    return unless @tracking_field
+    return unless @query_executor.is_a?(LogStash::Inputs::Elasticsearch::SearchAfter)
+
+    if @resolved_search_api != "search_after" || @response_type != "hits"
+      raise ConfigurationError.new("The `tracking_field` feature can only be used with `search_after` non-aggregation queries")
+    end
+
+    @tracking_field_seed ||= Time.now.utc.iso8601
```

Comment:
Would it make sense to default to, say, epoch, to default to catching as many messages as possible, rather than defaulting to skipping past events ingested prior to the first time this plugin is run?